api/candlepin-api-spec.yaml

```diff
@@ -193,6 +193,27 @@ paths:
               schema:
                 $ref: '#/components/schemas/ExceptionMessage'
 
+  /status:
+    get:
+      description: Returns status of the server
+      tags:
+        - Status
+      operationId: status
+      security: []
+      responses:
+        200:
+          description: ''
```

Comment:
Please remove this line

Reply:
@kahowell 
I am getting the following error after removing description from response section. 

`Execution failed for task ':api:openApiGenerate'.
> There were issues with the specification. The option can be disabled via validateSpec (Maven/Gradle) or --skip-validate-spec (CLI).
   | Error count: 1, Warning count: 0
  Errors: 
        -attribute paths.'/status'(get).responses.200.description is missing
`

Reply:
Ah, I should have looked closer! We'll need to fill in something for description to make the spec valid as you already figured out.

Reply:
For a 200, a simple description of "OK" would be fine.

Reply:
After some discussion, it would be better to include some description (here and in other PRs with empty descriptions like https://github.com/candlepin/candlepin/pull/2577) so that we can use this as an opportunity to improve our API documentation.

It doesn't have to be a lot, a verbose to-the-point description will do. This document has some nice guidelines/examples: https://swagger.io/docs/specification/describing-responses/

Reply:
Done.